packages/cli/src/commands/index.ts

```diff
@@ -22,6 +22,16 @@ export function registerCommands(program: CommanderStatic) {
     .command('app:build')
     .description('Build an app for a production release')
     .option('--stats', 'Write bundle stats to output directory')
+    .option(
+      '--parallel [value]',
+      'Enable multi-process parallel running and set number of concurrent runs',
```

Comment:
"Enable (true/false) or control amount of (number) parallelism in some build steps"